src/release/scripts/release.sh

```diff
@@ -4,12 +4,57 @@
 
 set -e
 
+intel_amd_tag=x86_64
```

Comment:
```suggestion
amd_tag=x86_64
```

Intel and amd mean the same thing in this context.

Reply:
Yeah I just wanted to be more specific but changed!